<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>run_window</class>
 <widget class="QWidget" name="run_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VIPCALs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="return_btn">
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="plots_btn">
     <property name="text">
      <string>Examine plots</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTextEdit" name="text_output"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
